keyboards/converter/ps2_usb/README.md

```diff
@@ -0,0 +1,51 @@
+PS/2 to USB keyboard converter
```

Comment:
Could you rename this file as `readme.md` please? As QMK is cross-platform, we'd like all filenames to be lowercase, because Windows gets silly about filenames that are perfectly fine on Mac and Linux.

Reply:
Done